### PR TITLE
Make README more actionable

### PR DESCRIPTION
**Pull request by will-cromar**

### CODE REVIEW

**Comment by JackCaoG** [comment]
I kind of like this pr, is this still a WIP?

**Comment by will-cromar** [comment]
I don't remember what else I was planning to write, so marking this ready for review.

File: README.md [hunk 1]
```diff
@@ -179,13 +178,51 @@ pip3 install torch_xla[tpuvm]
 
 This is only required on Cloud TPU VMs.
 
-## Performance Profiling and Auto-Metrics Analysis
```
Comment (JackCaoG) [inline README.md:182]: why is Performance Profiling section remode?

Comment (will-cromar) [reply README.md:182]: I can move the link to the Cloud documentation with the rest of the tutorials. The colab notebook is deprecated, and I'm not sure the other two links are actually instructive.

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**